library/iojs

```diff
@@ -2,15 +2,24 @@
 
 1.0.4: git://github.com/iojs/docker-iojs@39ab80a90ff9addd935d3acf1f5bc3eff49d6ffc 1.0
```

Comment:
I would like the "Supported tags" section to be accurate.  If you promise a
PR for that will come soon, I'm OK with this one as-is for now. :+1: